cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -363,20 +368,20 @@ private void restartIfNecessary(PodRef podRef, RestartContext restartContext)
             checkReconfigurability(podRef, pod, restartContext);
             if (restartContext.forceRestart || restartContext.needsRestart || restartContext.needsReconfig) {
                 if (!restartContext.forceRestart && deferController(podRef, restartContext)) {
-                    LOGGER.debugCr(reconciliation, "Pod {} is controller and there are other pods to roll", podRef);
-                    throw new ForceableProblem("Pod " + podRef.getPodName() + " is currently the controller and there are other pods still to roll");
+                    LOGGER.debugCr(reconciliation, "Pod {} is controller and there are other pods to verify. Will favor trying to verify non-controller pods first", podRef);
```

Comment:
I would probably leave the `Will favor trying to verify non-controller pods first` part out. But if you wanna keep it, I would put it more as an announcement. It does not really matter what the operator favors as there is no choice. 

Also, If you use multiple sentences, it would be good to finish both of them with `.`.

Reply:
Done.

The `.` removal was a mistake, I meant to remove it from the Exception message.